frontend_scv/lib/screens/view_assignment_screen.dart

```diff
@@ -22,31 +18,19 @@ class ViewAssignmentScreen extends StatefulWidget {
 enum ConditionType { Normal, Payment, Duration }
 
 class _ViewAssignmentScreenState extends State<ViewAssignmentScreen> {
-  // final _conditionTitleController = TextEditingController();
-  // final _conditionDescriptionController = TextEditingController();
-  // final _paymentConditionAmountController = TextEditingController();
-  // final _durationConditionAmountController = TextEditingController();
   var _isLoading = false;
-  // var _isInit = true;
-
   NegotiationService negotiationService = NegotiationService();
-
-  GlobalKey<FormState> _formKey = GlobalKey<FormState>();
-
   @override
   void initState() {
-     //print('InitState()');
     super.initState();
   }
 
   Widget build(BuildContext context) {
-    //final contractId = ModalRoute.of(context).settings.arguments as String;
-    final testCon = ModalRoute.of(context).settings.arguments as Contract;
-    final loadedContract = testCon;
-
+    final loadedContract =
+        ModalRoute.of(context).settings.arguments as Contract;
     return Scaffold(
       appBar: AppBar(
-        title: Text(loadedContract.title),
+        title: FunkyText('Current Assignment'),
```

Comment:
Why is it called funkyText

Reply:
GOT FUNK?
![image](https://user-images.githubusercontent.com/41339070/133856249-f38ca9c1-83e0-4761-a731-657893f53160.png)